### DISCUSSION TITLE
How to get the fancy visuals in paraview

### DISCUSSION THREAD

**Discussion by minzhangut** [PMEAL/porespy#682] [Q&A] [answered]
I am wondering how to get the last fancy visual (balls and sticks) in this example using paraview: https://porespy.org/examples/networks/tutorials/snow_advanced.html. I saved my network as vtp file, and when I opened it using paraview, I could only see the lines.

**Comment by jgostick** [comment, marked as answer]
You can load both the TIFF image and the VTP file.  They need to be scaled and (sometimes) rotated to match each other.  For the TIFF image you should apply a threshold filter to only show pixels labeled 0 (i.e. solid), then the network will be nestled in the void space.  There is a tutorial [here](https://github.com/PMEAL/OpenPNM/blob/dev/examples/getting_started/view_vtp_in_paraview.md) on how to get the network looking good in paraview.  The 'fancy' part requires turning on ray tracing, which is an option at the bottom of the left panel....but you need decent computer to handle it.

**Reply by minzhangut** [reply]
Hello Prof. Gostick, thanks for the quick reply! I am wondering how to rotate the .tif file?

**Reply by jgostick** [reply]
numpy.swapaxis, lr, up all do that.  We have a function in porespy called align_image_with_openpnm that does it too.

**Reply by minzhangut** [reply]
Got it! Thanks Prof. Gostick!
